app/jobs/application_job.rb

```diff
@@ -38,6 +38,7 @@ def deserialize(job_data)
     case queue_name.to_sym
     when :asap then -1000
     when :default, :automation then 0
+    when :upvs_sync then 500
```

Comment:
tu by som mozno netahal domenu "upvs" mozeme to nazvat `sync` ?

Reply:
queue pre UPVS sync alebo pre tie menej dolezite syncy?

Reply:
tie menej dolezite syncy mi pridu ze by mohli byt aj later ci?